lib/genpack.js

```diff
@@ -10,6 +10,12 @@ const emojiComponents = require('unicode-emoji-json/data-emoji-components')
 const keywordsMap = {}
 const emmojiInfo = {}
 
+const getCodePoint = (char) => {
+  const hex = char.codePointAt(0).toString(16).toUpperCase()
+  const leadingZeroes = hex.length < 4 ? '0'.repeat(4 - hex.length) : ''
+  return leadingZeroes + hex
+}
+
```

Comment:
Let's move this to the bottom of the file and make it a function declaration instead of a function expression. We can also use [padStart](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart) to format the string.